src/zh/zaimanhua/src/eu/kanade/tachiyomi/extension/zh/zaimanhua/Zaimanhua.kt

```diff
@@ -132,7 +132,7 @@ class Zaimanhua : HttpSource(), ConfigurableSource {
 
     // path: "/comic/detail/mangaId"
     override fun mangaDetailsRequest(manga: SManga): Request =
-        GET("$apiUrl/comic/detail/${manga.url}?channel=android", apiHeaders)
+        GET("$apiUrl/comic/detail/${manga.url}?channel=android&_v=2.2.5", apiHeaders)
```

Comment:
```suggestion
        GET("$apiUrl/comic/detail/${manga.url}?_v=2.2.5", apiHeaders)
```

From my test, `channel=android` is not necessary and the app is now using `platform=android`